src/app/modules/main/chat_section/chat_content/users/module.nim

```diff
@@ -154,29 +140,39 @@ method loggedInUserImageChanged*(self: Module) =
   self.view.model().setIcon(singletonInstance.userProfile.getPubKey(), singletonInstance.userProfile.getIcon(),
   singletonInstance.userProfile.getIsIdenticon())
 
+method addChatMember*(self: Module,  id: string) =
```

Comment:
This is just the code from `onChatMembersAdded` that I moved out to this function so I can reduce duplication.